app/src/main/java/com/github/se/wanderpals/ui/screens/trip/finance/FinancePieChart.kt

```diff
@@ -27,45 +30,48 @@ fun FinancePieChart(
     expenses: List<Expense>,
     radiusOuter: Dp = 100.dp,
     chartBandWidth: Dp = 20.dp,
+    totalValueDisplayIsEnabled: Boolean = false
```

Comment:
Missing @ param comment for the new parameter

Reply:
Thank you for noticing it !